README.md

```diff
@@ -53,6 +53,16 @@ The `graphqlHTTP` function accepts the following options:
   * **`graphiql`**: If `true`, may present [GraphiQL][] when loaded directly
     from a browser (a useful tool for debugging and exploration).
 
+  * **`loadPersistedDocument`**: A function that takes an input id and returns a
+    valid Document. If provided, this will allow your GraphQL endpoint to execute
+    a document specified via `documentID`.
```

Comment:
While the document might have a unique name like `HomePageQuery`, this is not enough, as the query likely changes over time as new client versions are released. It is important that old clients still get the old version of `HomePageQuery`.